<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationDirectoryDialog</class>
 <widget class="QDialog" name="CalibrationDirectoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Calibration Folder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonBright">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="MQIT/Resources.qrc">
         <normaloff>:/Resources/Icon/icons8-images-folder-48.png</normaloff>:/Resources/Icon/icons8-images-folder-48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDirBright">
       <property name="placeholderText">
        <string>Bright Calibration Directory</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNameBright">
       <property name="text">
        <string>Bright.tiff</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>File Name</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBrightCnt">
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBrightStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="MQIT/Resources.qrc">
         <normaloff>:/Resources/Icon/icons8-camera-48.png</normaloff>:/Resources/Icon/icons8-camera-48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonDark">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="MQIT/Resources.qrc">
         <normaloff>:/Resources/Icon/icons8-images-folder-48.png</normaloff>:/Resources/Icon/icons8-images-folder-48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDirDark">
       <property name="placeholderText">
        <string>Dark Calibration Directory</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNameDark">
       <property name="text">
        <string>Dark.tiff</string>
       </property>
       <property name="placeholderText">
        <string>File Name</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxDarkCnt">
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDarkStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="MQIT/Resources.qrc">
         <normaloff>:/Resources/Icon/icons8-camera-48.png</normaloff>:/Resources/Icon/icons8-camera-48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonAverage">
     <property name="text">
      <string>Generate Average Image</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MQIT/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
